### ISSUE TITLE
Docstring for org-journal-date-prefix provides conflicting advice

### ISSUE THREAD

**Issue by doolio** [bastibe/org-journal#249]
**Describe the bug**
I think the docstring for `org-journal-date-prefix` provides confusing/conflicting advice. It states:
```
By default, this is a org-mode heading. Another good idea would be
"#+TITLE: " for org titles.
```
I presume this applies only to daily journals as it then goes on to state:
```
Setting org-journal-date-prefix to something other than "* "
for weekly/monthly/yearly journal files won't work correctly.
```
This was discussed in issue #150 and updated as part of commit [104c469](https://github.com/bastibe/org-journal/commit/104c46958c2e99ae3ebec18763e046303174214f). Perhaps the user should be pointed towards `org-journal-file-header` instead if they wish to include `#+TITLE:` as demonstrated in issue #244?

[Aside: I thought there was a shift to prefer lowercase keywords (e.g. `#+title`) in Org with this [commit](https://code.orgmode.org/bzg/org-mode/commit/13424336a6f30c50952d291e7a82906c1210daf0)]

**Expected behavior**
Docstring should be clear and unambigouos.

**Desktop environment**
 - OS: `Parabola GNU/Linux`
 - Emacs Version: `GNU Emacs 26.3 (build 1, i686-pc-linux-gnu, GTK+ Version 3.24.20)
 of 2020-05-01`
- Org Version: `9.3.6`
- `org-journal` commit: `2c43b10eed0659f8e47797e5e53d2973f939284d` 

Happy to submit a PR for consideration if you agree?

**Comment by cslux** [comment]
> I think the docstring for `org-journal-date-prefix` provides confusing/conflicting advice. It states:
> 
> ```
> By default, this is a org-mode heading. Another good idea would be
> "#+TITLE: " for org titles.
> ```
> 
> I presume this applies only to daily journals as it then goes on to state:

Exactly, this applies only to daily journal files. I don't like it, because it limits the possibilities what can be done greatly.
  

> This was discussed in issue #150 and updated as part of commit [104c469](https://github.com/bastibe/org-journal/commit/104c46958c2e99ae3ebec18763e046303174214f). Perhaps the user should be pointed towards `org-journal-file-header` instead if they wish to include `#+TITLE:` as demonstrated in issue #244?

Good idea. 

> [Aside: I thought there was a shift to prefer lowercase keywords (e.g. `#+title`) in Org with this [commit](https://code.orgmode.org/bzg/org-mode/commit/13424336a6f30c50952d291e7a82906c1210daf0)]

I missed that. We should point that out too, and update the documentation accordingly.

I PR which resolves the ambiguousness would be create. Thank you!

**Comment by cslux** [comment]
Thank you for your PR @doolio. Next time use "Fixes #249" etc. https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords
There should be no word between keyword and the issue number.

**Comment by doolio** [comment]
> There should be no word between keyword and the issue number.

Understood, and thanks for the help reference.
